File: Diving into JavaScript: For Beginners/step02/2.06/2.6.01.js
/*

Запросите у пользователя число x, а затем число y. Проверьте является ли х больше чем у. 
Выведите результат проверки (true или false). 

Sample Input 1:

8
11

Sample Output 1:

false

Sample Input 2:

65
15

Sample Output 2:

true

Sample Input 3:

5
5

Sample Output 3:

false

*/

let x = Number(prompt());
let y = Number(prompt());

let isGreater = x > y;
console.log(isGreater);

File: Python Advanced: An advanced course for beginners/2.06/2.6.01.py
# Список предметов
lst = ['нож', 'пистолет', 'фонарь', 'крест', 'рукоять от меча']

# Далее Ваш код
a, b, c = input(), input(), input()
if a not in lst:
    lst.append(a)
if b not in lst:
    lst.append(b)
if c not in lst:
    lst.append(c)

print('Снаряжение обновлено:',lst)
